src/FullTextSearchHelpers.ts

```diff
@@ -10,6 +10,7 @@ import _ = require('lodash');
 import SyncTasks = require('synctasks');
 
 import NoSqlProvider = require('./NoSqlProvider');
+import { ItemType } from './NoSqlProvider';
```

Comment:
Isn't this NoSqlProvider.ItemType (already imported)

Reply:
Yeah I just don't want to have to type out the whole thing every time.